aries_cloudagent/protocols/issue_credential/v2_0/manager.py

```diff
@@ -420,31 +430,33 @@ async def receive_request(
                     cred_request_message._thread_id,
                     role=V20CredExRecord.ROLE_ISSUER,
                 )
-            except StorageNotFoundError:
+            except StorageNotFoundError as ex:
                 # holder sent this request free of any offer
-                cred_ex_record = V20CredExRecord(
-                    connection_id=connection_id,
-                    thread_id=cred_request_message._thread_id,
-                    initiator=V20CredExRecord.INITIATOR_EXTERNAL,
-                    role=V20CredExRecord.ROLE_ISSUER,
-                    auto_remove=not self._profile.settings.get(
-                        "preserve_exchange_records"
-                    ),
-                    trace=(cred_request_message._trace is not None),
-                    auto_issue=self._profile.settings.get(
-                        "debug.auto_respond_credential_request"
-                    ),
-                )
+                if len(format_handlers_without_offer):
```

Comment:
Nitpick but I'd say it's more idiomatic to just do `if format_handlers_without_offer:`